tools/generate-svg-from-all-fbps

```diff
@@ -56,11 +56,18 @@ function die() {
 [ -e $FBP_TO_DOT ] || die "Couldn't find sol-fbp-to-dot."
 
 for f in $(find src/ -name \*.fbp); do
-    if [ -f ${f/.fbp/-expected-fail.txt} ] ||
-       [ -f ${f/.fbp/-expected-fail.re.txt} ]; then
-	continue
+    # skip error tests
+    if grep "TEST-EXPECTS-ERROR" ${f} > /dev/null; then
+       continue
     fi
+
     dir=$(dirname $f)
     mkdir -p $OUTPUT/$dir
+
+    # Copy fbp used to generate svg removing license.
+    # It's going to be used to display code snippets, having 30 lines
+    # of license is annyoing.
+    tail -n +31 $f > $OUTPUT/$f
```

Comment:
I understand, it's a weak solution.
But since we're using a script / template / coping and pasting a header with this size, it seems to work.